net/tstun/wrap.go

```diff
@@ -692,6 +692,27 @@ func (t *Wrapper) SetFilter(filt *filter.Filter) {
 	t.filter.Store(filt)
 }
 
+// InjectInboundDirect makes the Wrapper device behave as if a packet
```

Comment:
InjectOutboundDirect is the wrong name, should be InjectInboundPacketBuffer.

I still don't like these "Inbound" and "Outbound" relative descriptors that don't tell you what they're relative to, but I see you're doing this for consistency with other similar helpers, LGTM.